src/components/ik-controller.js

```diff
@@ -168,9 +168,10 @@ AFRAME.registerComponent("ik-controller", {
     }
 
     const root = this.ikRoot.el.object3D;
+    root.updateMatrices();
     const { camera, leftController, rightController } = this.ikRoot;
 
-    camera.object3D.updateMatrix();
```

Comment:
for this to be semantically the same, do you need to flip `matrixNeedsUpdate = true`? iirc `updateMatrix` forces a local matrix update.